src/scenes/sneaky_spirits.inc.c

```diff
@@ -7,25 +8,30 @@
 // For readability. !TODO - CHANGE/REMOVE
 #define gSneakySpiritsInfo D_030055d0->gameInfo.sneakySpirits
 
+// Game-Specific Macros:
+#define VERSION_GAME_1 0
+#define VERSION_REMIX 1
+#define VERSION_GAME_2 2
```

Comment:
These macro names are pretty dangerous since they'd carry into other code. Maybe like VERSION_SNEAKY_SPIRITS_1, VERSION_SNEAKY_SPIRITS_2, VERSION_SNEAKY_SPIRITS_REMIX, idk just something less general